fmcapi/fmc.py

```diff
@@ -281,7 +281,7 @@ def send_to_api(
                     more_items=self.more_items,
                 )
             else:
-                json_response["items"] += self.more_items
+                json_response["items"] = self.more_items + json_response["items"]
```

Comment:
I just assumed that was an FMC idiosyncrasy and dealt with it in code. Nice find.